activiti-api-process-model/src/main/java/org/activiti/api/process/model/events/BPMNTimerEvent.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2018 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.api.process.model.events;
+
+import org.activiti.api.model.shared.event.RuntimeEvent;
+import org.activiti.api.process.model.BPMNTimer;
+
+public interface BPMNTimerEvent extends RuntimeEvent<BPMNTimer, BPMNTimerEvent.TimerEvents> {
+
+    enum TimerEvents {
+        TIMER_SCHEDULED,
+        TIMER_FIRED,
+        JOB_CANCELED,
+        JOB_EXECUTION_SUCCESS,
+        JOB_EXECUTION_FAILURE,
+        JOB_RETRIES_DECREMENTED
```

Comment:
Let's hide the notion of job in the event names: `TIMER_CANCELED`, `TIMER_EXECUTED`, TIMER_FAILLED, TIMER_RETRIES_DECREMENTED. Does it make sense?

Reply:
I would keep this to be consistent with 
https://github.com/Activiti/Activiti/blob/c8666252c669d89f2c6d7063e4a89aadf6b73175/activiti-engine/src/main/java/org/activiti/engine/delegate/event/ActivitiEventType.java
Does this make sense?

Reply:
I see what you mean. My only concern is that job is an internal concept that I’d prefer to do not expose. These are implementation details. We have already some names that are different when we are mapping generic events like ENITITY_CREATED, ENTITY_UPDATE, etc.

Reply:
Implemented here: https://github.com/Activiti/activiti-api/pull/128/commits/5f329dade3ddf0fbc0bfe012c62ca55a465909b2